packages/react-css-utilities/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@shopify/react-css-utilities",
+  "version": "0.0.0",
+  "license": "MIT",
+  "description": "A set of react styling related utilities.",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "scripts": {
+    "build": "tsc --p tsconfig.build.json",
+    "prepublishOnly": "yarn run build"
+  },
+  "publishConfig": {
+    "access": "public",
+    "@shopify:registry": "https://registry.npmjs.org"
+  },
+  "author": "Shopify Inc.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Shopify/quilt.git"
+  },
+  "bugs": {
+    "url": "https://github.com/shopify/quilt/issues"
+  },
+  "homepage": "https://github.com/Shopify/quilt/blob/master/packages/react-css-utilities/README.md",
+  "dependencies": {
+    "classnames": "^2.2.6"
+  },
+  "devDependencies": {
+    "@types/classnames": "^2.2.7",
```

Comment:
The types actually need to be a dependency here, otherwise the consumer won't necessarily have them installed.